CHANGES.rst

```diff
@@ -3,6 +3,15 @@ Change Log
 
 Unreleased
 ----------
+**Added**
+
+* :meth:`~.Emoji.update` supports modifying an emoji's permissions.
+* :meth:`~.SubredditEmoji.add` now supports optionally passing booleans to set
+  an emoji's permissions upon upload.
+
+* :meth:`~.Emoji.update` supports modifying an emoji's permissions.
+* :meth:`~.SubredditEmoji.add` now supports optionally passing booleans to set
+  an emoji's permissions upon upload.
```

Comment:
This got duplicated, so remove an entry.

Reply:
Good catch, yep.